### PR TITLE
[JENKINS-29653] Added a little vertical space between  button and the textarea

### PR DESCRIPTION
**Pull request by recena**
[JENKINS-29653](https://issues.jenkins-ci.org/browse/JENKINS-29653)

workflow

@reviewbybees

### CODE REVIEW

**Comment by jglick** [comment]
:bee:

**Comment by jtnord** [comment]
:bee:

**Comment by recena** [comment]
@reviewbybees done

File: cps/src/main/resources/org/jenkinsci/plugins/workflow/cps/Snippetizer/block.jelly [hunk 1]
```diff
@@ -61,7 +61,7 @@ THE SOFTWARE.
         <j:set var="id" value="${h.generateId()}"/>
         <f:block>
             <input type="button" value="Generate Groovy" onclick="handlePrototype_${id}(); return false" class="submit-button primary"/>
-            <f:textarea id="prototypeText_${id}" readonly="true"/>
+            <f:textarea id="prototypeText_${id}" readonly="true" style="margin: 10px 0px;" />
```
Comment (KostyaSha) [inline cps/src/main/resources/org/jenkinsci/plugins/workflow/cps/Snippetizer/block.jelly]: just add `<br/>` before? Style for textarea may be controlled and changed in core.

**Comment by recena** [comment]
@jglick @jtnord I've review the CSS style because only `margin-top` necessary.

**Comment by KostyaSha** [comment]
@recena maybe check with @tfennelly why such issue exists?

File: cps/src/main/resources/org/jenkinsci/plugins/workflow/cps/Snippetizer/block.jelly (hunk 1 above)
Comment (recena) [inline cps/src/main/resources/org/jenkinsci/plugins/workflow/cps/Snippetizer/block.jelly]: @KostyaSha http://www.w3.org/TR/html-markup/br.html, specially here this section **Permitted parent elements**.

**Comment by recena** [comment]
@KostyaSha Any opinion is welcome, specially from @tfennelly

**Comment by recena** [comment]
@KostyaSha When Jenkins Core provides a UI toolkit more flexible, semantized, non table-based layout, etc... then, I'll review this PR again.

File: cps/src/main/resources/org/jenkinsci/plugins/workflow/cps/Snippetizer/block.jelly (hunk 1 above)
Comment (KostyaSha) [inline cps/src/main/resources/org/jenkinsci/plugins/workflow/cps/Snippetizer/block.jelly]: maybe try wrap with `f.entry() { <f:textarea ... }` that is common pattern in jenkins jelly? AFAIR in comparison to block in adds small indentation vertically

**Comment by KostyaSha** [comment]
@recena btw, for any UI changes always attach 2 screenshots: before and after

**Comment by KostyaSha** [comment]
@recena they can do table div migration for generated jelly codes, when you hacking styles you may end with issues as soon as somebody changed styles in core/jelly. I already had such fun with layous in build-pipeline plugin. Though for buttons it shouldn't be so much...

**Comment by jglick** [comment]
:bee: (previous bees reset by apparent rebasing)

**Comment by recena** [comment]
@jglick Yes, I reviewed the CSS style.

**Comment by jtnord** [comment]
:bee: 

@KostyaSha whilst there is likely an issue in the Jenkins style sheet somewhere, getting that fixed will require an updated Jenkins and a bump in the workflow dependency which will delay getting this usability issue out to users.  (at present it looks like a tab header rather than a button).
A tweak would likely be needed when the root cause is fixed in Jenkins core - but at worst it will produce an ugly layout in the interim (which is exactly what this is attempting to to a quick fix of)  but that is better than locking users out by requiring a latest and greatest version of core.


Pre fix screenshot

**Comment by recena** [comment]
@reviewbybees done

**Pull request merged**